magpie/register.py

```diff
@@ -25,6 +49,11 @@
 SERVICES_PHOENIX = 'PHOENIX'
```

Comment:
Is this file only related to Phoenix? Because all Phoenix related stuff could be marked as deprecated... and removed as soon as the latest twitcher version is used.

Reply:
Indeed, all of Phoenix related code is preserved for back compatibility to old twitcher, but is mostly deprecated now.